### DISCUSSION TITLE
How to setup temperature sensor for Echo ?

### DISCUSSION THREAD

**Discussion by Rayeliint** [alandtse/alexa_media_player#1699] [Q&A]
Can someone please point me in the direction or explain how to setup a temperature sensor for my Echo Plus in Home Assistant?

Looked everywhere and cant seem to find how to do it.

**Comment by pvillanyi** [comment]
Funny, no one answered...
I would be also interested

**Comment by chickentwo** [comment]
go into integrations, Alexa Media Player, click on CONFIGURE then click in the box for "Include devices connected via Echo"
